aspnet/tutorials/first-mvc-app/adding-model.rst

```diff
@@ -1,27 +1,24 @@
 Adding a model
 ==================================================
-
 By `Rick Anderson`_
 
+
 In this section you'll add some classes for managing movies in a database. These classes will be the "**M**\odel" part of the **M**\VC app.
 
-You’ll use a .NET Framework data-access technology known as the `Entity Framework <http://ef.readthedocs.org/>`__ to define and work with these model classes. The Entity Framework (often referred to as EF) supports a development paradigm called *Code First*. Code First allows you to create model objects by writing simple classes. (These are also known as POCO classes, from "plain-old CLR objects.") You can then have the database created on the fly from your classes, which enables a very clean and rapid development workflow. If you are required to create the database first, you can still follow this tutorial to learn about MVC and EF app development. 
+You’ll use a .NET Framework data-access technology known as the `Entity Framework <http://ef.readthedocs.org/>`__ to define and work with these data model classes. Entity Framework (often referred to as **EF**) features a development paradigm called *Code First*. You write the code first, and the database tables are created from this code. Code First allows you to create data model objects by writing simple classes. (These are also known as POCO classes, from "plain-old CLR objects.") The database is created from your classes. If you are required to create the database first, you can still follow this tutorial to learn about MVC and EF app development. 
```

Comment:
I would probably stick with EF Core for the moment, to help distinguish between EF6.x. In our docs we typically do "Entity Framework Core (EF Core)" the first time, and then use the acronym from that point on.